.github/workflows/builds.yml

```diff
@@ -261,7 +261,7 @@ jobs:
           CSC_IDENTITY_AUTO_DISCOVERY: false
           SNAPCRAFT_LOGIN: ${{ secrets.SNAPCRAFT_LOGIN }}
         run: |
-          sudo snap install snapcraft --classic
+          sudo snap install snapcraft --channel=5.x/stable --classic
```

Comment:
it should be v6.1 as per this successful build https://github.com/ferdium/ferdium-app/runs/6872922102?check_suite_focus=true#step:16:20

Reply:
V6 isnt listed at [https://snapcraft.io/snapcraft](https://snapcraft.io/snapcraft)
![image](https://user-images.githubusercontent.com/36506137/173797074-df91bebe-5300-437b-aff3-591a72808420.png)

Reply:
> V6 isnt listed at https://snapcraft.io/snapcraft ![image](https://user-images.githubusercontent.com/36506137/173797074-df91bebe-5300-437b-aff3-591a72808420.png)

Weird :o

@NathanaelGandhi , by any chance do you have time to adapt our build to 7.x today?

Reply:
Yes - this page doesn't list any `6.x` - thus, I left it at `5.x`